pkg/registry/apis/scope/register.go

```diff
@@ -0,0 +1,89 @@
+package scope
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+	"k8s.io/apiserver/pkg/registry/generic"
+	"k8s.io/apiserver/pkg/registry/rest"
+	genericapiserver "k8s.io/apiserver/pkg/server"
+	"k8s.io/kube-openapi/pkg/common"
+
+	scope "github.com/grafana/grafana/pkg/apis/scope/v0alpha1"
+	"github.com/grafana/grafana/pkg/services/apiserver/builder"
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
+)
+
+var _ builder.APIGroupBuilder = (*ScopeAPIBuilder)(nil)
+
+// This is used just so wire has something unique to return
+type ScopeAPIBuilder struct{}
+
+func NewScopeAPIBuilder() *ScopeAPIBuilder {
+	return &ScopeAPIBuilder{}
+}
+
+func RegisterAPIService(features featuremgmt.FeatureToggles, apiregistration builder.APIRegistrar) *ScopeAPIBuilder {
+	if !features.IsEnabledGlobally(featuremgmt.FlagGrafanaAPIServerWithExperimentalAPIs) {
+		return nil // skip registration unless opting into experimental apis
+	}
+	builder := NewScopeAPIBuilder()
+	apiregistration.RegisterAPI(NewScopeAPIBuilder())
```

Comment:
should this be using the builder variable?
```suggestion
	apiregistration.RegisterAPI(builder)
```